JSDemos/Demos/DataGrid/ExportToPDF/description.md

```diff
@@ -0,0 +1,5 @@
+Our [DataGrid](/Documentation/ApiReference/UI_Widgets/dxDataGrid/) allows you to easily and accurately export its contents to a PDF file. To enable the export operation, you must reference or import <a href="https://github.com/MrRio/jsPDF" target="_blank">jsPDF</a> library.
```

Comment:
'easily and accurately' - taken from Ray's description? 

I think you need an article here: "the jsPDF library". It differs when the usage below, where you don't add the word "library".

Reply:
@Abadzhev Hello again, I attached the screenshot with a similar description rewritten by Ray.

![image](https://user-images.githubusercontent.com/33040934/95055183-99185a00-06fb-11eb-8cad-3369cfb3667a.png)

Reply:
Okay - not going to argue with that :-)